emails/models.py

```diff
@@ -100,6 +100,12 @@ def at_max_free_aliases(self):
     def fxa(self):
         return self.user.socialaccount_set.filter(provider='fxa').first()
 
+    @property
+    def display_name(self):
+        # if display name is not set on FxA the
+        # displayName key will not exist on the extra_data
+        return self.fxa.extra_data.get('displayName')
```

Comment:
should we make this return the email address if `displayName` doesn't exist?

```suggestion
        return self.fxa.extra_data.get('displayName', self.fxa.extra_data.email)
```

Reply:
@maxxcrawford, do you want the backend to return the email address for the display name var in the context if the display name doesn't exist or just return `None`?

Reply:
@say-yawn Great questions – Lemme check with `@efeo`

Reply:
Any update here?

Reply:
Update: If they don't have a display name set, we're just going to say "Welcome"…so `none` should work. As long as we can wrap up the logic there. I'm happy to impliment that portion after this lands with the name.